packages/integration/lib/src/views/feed/universal_feed/feed_screen.dart

```diff
@@ -8,6 +8,7 @@ import 'package:flutter/services.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:likeminds_feed_flutter_core/likeminds_feed_core.dart';
 import 'package:likeminds_feed_flutter_core/src/utils/web/feed_web_configuration.dart';
+import 'package:likeminds_feed_flutter_core/src/views/post/post_detail_screen.dart';
```

Comment:
How is this not part of core library import, otherwise it should be part of `views.dart`